app/javascript/components/Greet.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+function Greet() {
```

Comment:
Nice! You created the Greeting component 💪

Reply:
Ok 👍🏽